articles/index.md

```diff
@@ -68,37 +68,37 @@ Develop a [theory of change](/theory) for your activist organisation using our t
 
 ## We start the activist movements of tomorrow
 
 Activist Handbook is here for all progressive, green and social activists. We empower people around the globe to reclaim the streets and uproot the system. For each cause, we have written a guide to help you get started.
 
 - **❤️ [Social activism](campaigns/social-activism)**: Learn how you can take action to put social challanges on the agenda and make a change. No matter whether you want to fight poverty or stand up for queer rights, we provide you with the tools and training to organise succesful campaigns.
-- **🌍 [Climate activism](/campaigns/climate-activism)**: Stand up for climate justice by learning from environmental grassroots movements such as Fridays for Future, Greenpeace and Extinction Rebellion. Learn how why protesting is effective and find a climate group to join. Or start your own initiative!
+- **🌍 [Climate activism](/campaigns/climate-activism)**: Stand up for climate justice by learning from environmental grassroots movements such as Fridays for Future, Greenpeace, and Extinction Rebellion. Learn how why protesting is effective and find a climate group to join. Or start your own initiative!
 - **🗳 [Political activism](/campaigns/political-activism)**: Learn everything about political activism. We have guides about door-to-door canvassing, preparing election campaign strategies, crowdfunding, running as a candidate, and many other forms of civic engagement.
-- **👩‍⚖️ [Human rights activism](/campaigns/human-rights)**: Find out how to defend human rights. Read our guides on how to stand up for justice by starting a campaign to fight for democracy, equality and freedom.
+- **👩‍⚖️ [Human rights activism](/campaigns/human-rights)**: Find out how to defend human rights. Read our guides on how to stand up for justice by starting a campaign to fight for democracy, equality, and freedom.
 - **📢 [Youth activism](/campaigns/youth-activism)**: We help students around the world make their voices heard. Read our guides to become more effective as a young change-maker and learn from examples of other succesful youth movements.
 - **🙌 [Community activism](/campaigns/community-activism)**: Make a difference in your community by organising locally. We explain what a community activist is and how to become involved.
 
 _Also make sure to check out the rest of our [campaigns](/campaigns)._
 
 <action-custom buttonlink="/join" buttonlabel="Join the campaign">
-  <h2>We're training 350 million activists</h2>
+  <h2>We need to train 350 million activists</h2>
```

Comment:
I am not sure about this change. I wanted to remove the short form, and then I felt like the "We are training 350M people" was claiming that we are actively doing it, all 350M at this moment. Which, might seems as claim from our side, which of course is not true, which I felt like, oh nice, a website trying to help us, but it is clearly misleading. Rather lacking in trust, but idk. It was simply not consistent with the other banners which suddenly say that we need to train 350M people.

Reply:
Changed it!